diff --git a/ld64/src/ld/OutputFile.cpp b/ld64/src/ld/OutputFile.cpp
index f9e5cf0..8db33db 100644
--- a/ld64/src/ld/OutputFile.cpp
+++ b/ld64/src/ld/OutputFile.cpp
@@ -4714,12 +4714,12 @@ void OutputFile::addClassicRelocs(ld::Internal& state, ld::Internal::FinalSectio
 		case ld::Fixup::kindStorePPCAbsHigh16:
 			{
 				assert(target != NULL);
-				if ( target->definition() == ld::Atom::definitionProxy ) {
-fprintf(stderr, "bad reloc target: %40s source %40s (offset 0x%" PRIx64 ") final addr 0x%" PRIx64 " atom offset: 0x%x [0x%" PRIx64 "]\n",
-                 target->name(), atom->name(), atom->objectAddress(), atom->finalAddress(),
-                 fixupWithStore->offsetInAtom, (uint64_t)atom->finalAddress()+fixupWithStore->offsetInAtom );
+//				if ( target->definition() == ld::Atom::definitionProxy ) {
+//fprintf(stderr, "bad reloc target: %40s source %40s (offset 0x%" PRIx64 ") final addr 0x%" PRIx64 " atom offset: 0x%x [0x%" PRIx64 "]\n",
+//                 target->name(), atom->name(), atom->objectAddress(), atom->finalAddress(),
+//                 fixupWithStore->offsetInAtom, (uint64_t)atom->finalAddress()+fixupWithStore->offsetInAtom );
 //					throwf("half word text relocs not supported in %s", atom->name());
-				}
+//				}
 				if ( _options.outputSlidable() ) {
 					if ( inReadOnlySeg )
 						noteTextReloc(atom, target);
