diff --git a/ld64/src/ld/InputFiles.cpp b/ld64/src/ld/InputFiles.cpp
index 10ba632..c6fcec3 100644
--- a/ld64/src/ld/InputFiles.cpp
+++ b/ld64/src/ld/InputFiles.cpp
@@ -273,7 +273,7 @@ ld::File* InputFiles::makeFile(const Options::FileInfo& info, bool indirectDylib
 				}
 			}
 			// if requested architecture is page aligned within fat file, then remap just that portion of file
-			if ( (fileOffset & 0x00000FFF) == 0 ) {
+			if ( (fileOffset & 0x00003FFF) == 0 ) {
 				// unmap whole file
 				munmap((caddr_t)p, info.fileLen);
 				// re-map just part we need
